src/netspresso_trainer/dataloaders/builder.py

```diff
@@ -100,7 +103,24 @@ def build_dataset(conf_data, conf_augmentation, task: str, model_name: str):
 def build_dataloader(conf, task: str, model_name: str, train_dataset, eval_dataset, profile=False):
 
     if task == 'classification':
-        collate_fn = None
+        if hasattr(conf.augmentation, 'mix_transforms'):
```

Comment:
@deepkyu 
I added `mixup` and `cutmix` as `mix_transforms`.
For consistency, how about change `recipe` in augmentation config to `transforms`?

Reply:
Agreed. Sounds better.